src/app/legacy/containers/PodcastPromo/Inline.jsx

```diff
@@ -31,37 +39,44 @@ const GEL_GROUP_1_WIDTH_260PX = '16.25rem';
 const GEL_GROUP_1_WIDTH_320PX = '20rem';
 const GEL_GROUP_1_WIDTH_360PX = '22.5rem';
 
+const standardMargin = `${GEL_SPACING_HLF} ${GEL_SPACING_DBL} ${GEL_SPACING_TRPL}`;
+
 const ResponsivePodcastPromoWrapper = styled.div`
-  ${({ dir }) => (dir === 'ltr' ? 'float: right;' : 'float: left;')}
-  background: ${props => props.theme.palette.LUNAR};
-  margin: ${GEL_SPACING_TRPL} 0;
+  ${({ dir }) => (dir === 'rtl' ? 'float: left;' : 'float: right;')}
+
+  margin: 0  ${GEL_SPACING} ${GEL_SPACING_TRPL};
   height: auto;
 
   @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) {
     width: 45%;
-    margin: ${GEL_SPACING_TRPL} ${GEL_SPACING_HLF};
+    margin: ${GEL_SPACING} ${GEL_SPACING} ${GEL_SPACING_TRPL};
   }
 
   @media (min-width: ${GEL_GROUP_1_WIDTH_260PX}) {
-    margin: ${GEL_SPACING_TRPL} ${GEL_SPACING};
+    margin: ${standardMargin};
+    margin-left: ${({ dir }) =>
+      dir === 'rtl' ? GEL_SPACING_DBL : GEL_SPACING_DBL};
   }
 
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    margin: ${GEL_SPACING_TRPL} ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: ${standardMargin};
+    margin-left: ${({ dir }) => (dir === 'rtl' ? '0' : GEL_SPACING_DBL)};
```

Comment:
Could we use `margin-inline` so we don't have to use the dir, it will stop duplication of css classes and less code https://github.com/bbc/simorgh/blob/1a8f8eb6bbb1a6e8b8e4342f177a323aee0a94ed/docs/Coding-Standards/Styles.stories.mdx#L104-L126
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline

Reply:
Wicked, will add it in.